### PR TITLE
[backport-14.1] UG-543 Verify Orchestration

### PR DESCRIPTION
**Pull request by dankolbrs**
Adds verification Orchestration service post upgrade which matches
existing test style. Also includes Orchestration in pre-upgrade
check.

(cherry picked from commit 69269b40cd47e48d89d5938f3be5ab9989aaaf47)

### CODE REVIEW

**Comment by dankolbrs** [comment]
recheck_cit_leapfrogupgrade

**Comment by evrardjp** [comment]
leapfrog upgrade isn't working yet. The rest is green so we should be good to go soon.

**Comment by mattt416** [comment]
Argh, this one needs a rebase @dankolbrs :(

**Comment by mattt416** [comment]
recheck_cit_swift

**Comment by mattt416** [comment]
recheck_cit_swift

**Comment by mattt416** [comment]
recheck_cit_swift

File: rpcd/playbooks/roles/rpc_pre_upgrade/templates/status.txt.j2 [hunk 1]
```diff
@@ -25,6 +25,11 @@ cinder service-list output as at {{ ansible_date_time.date }}-{{ ansible_date_ti
 {{ line }}
 {% endfor %}
 
+orchestration service list output as at {{ ansible_date_time.date }}-{{ ansible_date_time.time }}
```
Comment (evrardjp) [inline rpcd/playbooks/roles/rpc_pre_upgrade/templates/status.txt.j2]: my eyes are itching here too for the break of idempotency, but that's outside the scope of this patch

**Comment by dankolbrs** [comment]
recheck_cit_all

**Comment by dankolbrs** [comment]
Error trying to build in jenkins
```
Can't find a revision to build

Invalid branch supplied to git module? Extra refspec required? Eg PR ref?
Indication 1 ```

Closing/reopening to see if it helps

**Comment by dankolbrs** [comment]
recheck_cit_xenial_deploy_swift

**Pull request merged**